content/productivity/simultaneous-attack/plugin.js

```diff
@@ -211,4 +212,4 @@ class SimultaneousAttack {
     }
 };
 
-export default SimultaneousAttack;
+export default SimultaneousAttack;
```

Comment:
```suggestion
export default SimultaneousAttack;

```

nbitpick add newline for Github.
![Screenshot 2022-04-01 at 12 45 30](https://user-images.githubusercontent.com/155505/161248809-515f697f-b081-4db5-befb-1bd1b7a29ffe.png)